docs/src/pages/en/showcase.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Media Chrome Showcase
+description: Links to other resources for all things Media Chrome and the ideas behind Media Chrome
+layout: ../../layouts/MainLayout.astro
+---
+
+## Who's using Media Chrome
+
+Here are some projects and web applications that currently use Media Chrome:
+
+[Mux Player](https://mux.com/player). Mux's in-house video player uses Media Chrome under the hood. In addition to the linked product page, Mux Player is also [open source under Mux Elements](https://github.com/muxinc/elements/tree/main/packages/mux-player).
+
+<img src="/public/mux-player-media-chrome.png" width="500" alt="Mux Player built with Media Chrome" />
```

Comment:
This image is missing in the deploy preview

https://media-chrome-docs-git-fork-dylanjha-dj-docs-updates-mux.vercel.app/en/showcase